drivers/docker/network.go

```diff
@@ -0,0 +1,84 @@
+package docker
+
+import (
+	"fmt"
+
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/hashicorp/nomad/plugins/drivers"
+)
+
+const infraContainerImage = "gcr.io/google_containers/pause-amd64:3.0"
+const dockerNetSpecLabelKey = "docker_sandbox_container_id"
+
+func (d *Driver) CreateNetwork(allocID string) (*drivers.NetworkIsolationSpec, error) {
+	// Initialize docker API clients
+	client, _, err := d.dockerClients()
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to docker daemon: %s", err)
+	}
+
+	repo, _ := parseDockerImage(infraContainerImage)
```

Comment:
Check the error in case we make infraContainerImage configurable someday.

Reply:
This actually doesn't return an error. It splits the image name (assigned to `repo` here) and tag.